.github/workflows/go.yaml

```diff
@@ -0,0 +1,30 @@
+name: Lint
+
+on:
+  push: {}
+  pull_request: {}
+
+permissions:
+  contents: read
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+
```

Comment:
It would be nice to have a `-name` property to quickly figure out what it's doing. e.g `-name: Setup Go`

Reply:
Discussed offline — by default it shows the command, so it looks like
> Run actions/setup-go@v3

Or for shell commands:
> Run go build -v ./...

Nino thinks that's enough.